node/network/approval-distribution/src/lib.rs

```diff
@@ -1682,6 +1710,14 @@ impl ApprovalDistribution {
 				state.handle_new_blocks(ctx, metrics, metas, rng).await;
 			},
 			ApprovalDistributionMessage::DistributeAssignment(cert, candidate_index) => {
+				let _span = state
+					.spans
+					.get(&cert.block_hash)
+					.map(|span| span.child("distribute-assignment"))
```

Comment:
I think the name is misleading here as we first import and then distribute. Both operations are covered.

Reply:
Got spot, will relabel this.